.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same here, those file should not be checked into a pull request during a project collaboration, you should only `git add` the specific files needed for your feature